oauth2/handler.go

```diff
@@ -49,6 +56,18 @@ func (h *Handler) SetRoutes(r *httprouter.Router) {
 	r.GET(ConsentPath, h.DefaultConsentHandler)
 	r.POST(IntrospectPath, h.IntrospectHandler)
 	r.POST(RevocationPath, h.RevocationHandler)
+	r.POST(WellKnownPath, h.WellKnownHandler)
```

Comment:
Yes absolutely, I'll also write tests for it, just wanted a branch to get started